genesyscloud/conversations_messaging_integrations_instagram/resource_genesyscloud_conversations_messaging_integrations_instagram_schema.go

```diff
@@ -0,0 +1,120 @@
+	regInstance.RegisterExporter(resourceName, ConversationsMessagingIntegrationsInstagramExporter())
+}
+
+// ResourceConversationsMessagingIntegrationsInstagram registers the genesyscloud_conversations_messaging_integrations_instagram resource with Terraform
+func ResourceConversationsMessagingIntegrationsInstagram() *schema.Resource {
+	return &schema.Resource{
+		Description: `Genesys Cloud conversations messaging integrations instagram`,
+
+		CreateContext: provider.CreateWithPooledClient(createConversationsMessagingIntegrationsInstagram),
+		ReadContext:   provider.ReadWithPooledClient(readConversationsMessagingIntegrationsInstagram),
+		UpdateContext: provider.UpdateWithPooledClient(updateConversationsMessagingIntegrationsInstagram),
+		DeleteContext: provider.DeleteWithPooledClient(deleteConversationsMessagingIntegrationsInstagram),
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		SchemaVersion: 1,
+		Schema: map[string]*schema.Schema{
+			`name`: {
+				Description: `The name of the Instagram Integration`,
+				Required:    true,
+				Type:        schema.TypeString,
+			},
+			`supported_content_id`: {
+				Description: `Reference to supported content profile associated with the integration`,
+				Optional:    true,
+				Type:        schema.TypeString,
+			},
+			`messaging_setting_id`: {
+				Description: `Messaging Setting for messaging platform integrations`,
+				Optional:    true,
+				Type:        schema.TypeString,
+			},
+			`page_access_token`: {
+				Description: `The long-lived Page Access Token of Instagram page. See https://developers.facebook.com/docs/facebook-login/access-tokens. When a pageAccessToken is provided, pageId and userAccessToken are not required.`,
+				Optional:    true,
+				Type:        schema.TypeString,
+			},
+			`user_access_token`: {
+				Description: `The short-lived User Access Token of Instagram user logged into Facebook app. See https://developers.facebook.com/docs/facebook-login/access-tokens. When userAccessToken is provided, pageId is mandatory. When userAccessToken/pageId combination is provided, pageAccessToken is not required.`,
+				Optional:    true,
+				Type:        schema.TypeString,
+			},
+			`page_id`: {
+				Description: `The page ID of Instagram page. The pageId is required when userAccessToken is provided.`,
+				Optional:    true,
+				Type:        schema.TypeString,
+				ForceNew:    true,
+			},
+			`app_id`: {
+				Description: `The app ID of Facebook app. The appId is required when a customer wants to use their own approved Facebook app.`,
+				Optional:    true,
+				Type:        schema.TypeString,
+				ForceNew:    true,
+			},
+			`app_secret`: {
+				Description: `The app Secret of Facebook app. The appSecret is required when appId is provided.`,
+				Optional:    true,
+				Type:        schema.TypeString,
+				ForceNew:    true,
+			},
```

Comment:
Are you sure all these fields need to be ForceNew? Did you see an error when you tried to update them ?